packages/malloy/src/lang/test/parse-expects.ts

```diff
@@ -37,11 +37,8 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace jest {
     interface Matchers<R> {
-      modelParsed(): R;
-      toBeErrorless(): R;
-      toCompile(): R;
-      modelCompiled(): R;
-      expressionCompiled(): R;
+      toParse(): R;
+      toTranslate(): R;
       toReturnType(tp: string): R;
       compileToFailWith(...expectedErrors: ErrorSpec[]): R;
```

Comment:
I think this name doesn't work — maybe `translateToFailWith`?

Reply:
I don't know how i missed this, yes `translationFailsWith` or somethiing